controller/src/beerocks/cli/beerocks_cli_socket.cpp

```diff
@@ -31,7 +31,7 @@ cli_socket::cli_socket(std::string temp_path_, std::string proxy_ip_)
     temp_path           = temp_path_;
 }
 
-cli_socket::~cli_socket() { disconnect(); }
+cli_socket::~cli_socket() { cli_socket::disconnect(); }
```

Comment:
This is exactly the same case as for cli_bml. I think it would be clearer if the two commits were squashed (nitpick).